### ISSUE TITLE
AttributeError: 'module' object has no attribute 'parse' -- failing tests

### ISSUE THREAD

**Issue by jlec** [statsmodels/statsmodels#2669]
```python
======================================================================
ERROR: statsmodels.graphics.tests.test_tsaplots.test_plot_month
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/lib64/python3.4/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/usr/lib64/python3.4/site-packages/numpy/testing/decorators.py", line 146, in skipper_func
    return f(*args, **kwargs)
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_4/lib/statsmodels/graphics/tests/test_tsaplots.py", line 39, in test_plot_month
    dta.index.values)
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_4/lib/statsmodels/compat/python.py", line 87, in lmap
    return list(map(*args, **kwargs))
  File "/var/tmp/portage/dev-python/statsmodels-0.6.1/work/statsmodels-0.6.1-python3_4/lib/statsmodels/graphics/tests/test_tsaplots.py", line 38, in <lambda>
    dates = lmap(lambda x : pd.datetools.parse('1 '+' '.join(x)),
AttributeError: 'module' object has no attribute 'parse'
```
statsmodels-0.6.1
numpy-1.10
pandas-0.17.0

**Comment by josef-pkt** [comment]
@jreback @jseabold 
I have no idea, I never looked at the date functions

It looks like it's the elnino date parsing/conversion
https://github.com/statsmodels/statsmodels/blob/master/statsmodels/graphics/tests/test_tsaplots.py#L50
but I don't even get it to run with python 3.4 and old pandas right now.

**Comment by josef-pkt** [comment]
In any case, it's just a problem with the test, not with the plot function itself

How do we get dates from this? That's what the test is trying to do.

```
>>> import statsmodels.api as sm
>>> dta = sm.datasets.elnino.load_pandas().data
>>> dta.head()
   YEAR    JAN    FEB    MAR    APR    MAY    JUN    JUL    AUG    SEP    OCT  \
0  1950  23.11  24.20  25.37  23.86  23.03  21.57  20.63  20.15  19.67  20.03   
1  1951  24.19  25.28  25.60  25.37  24.79  24.69  23.86  22.32  21.44  21.77   
2  1952  24.52  26.21  26.37  24.73  23.71  22.34  20.89  20.02  19.63  20.40   
3  1953  24.15  26.34  27.36  27.03  25.47  23.49  22.20  21.45  21.25  20.95   
4  1954  23.02  25.00  25.33  22.97  21.73  20.77  19.52  19.33  18.95  19.11   

     NOV    DEC  
0  20.02  21.80  
1  22.33  22.89  
2  20.77  22.39  
3  21.60  22.44  
4  20.27  21.30  
```

**Comment by yarikoptic** [comment]
I think this was fixed with f1dc8979b09bc1736149993f895943b3158ee2db and could be closed?

**Comment by josef-pkt** [comment]
@yarikoptic  thanks for the pointer
